musthe/musthe.py

```diff
@@ -84,6 +84,18 @@ def has_sharp(self):
         return self.name not in 'EB'
 
 
+class LetterSolfege(Letter):
+    """
+    Another tone naming.
+
+    There are still 7: Dó, Ré, Mi, Fá, Sol, Lá, and Si.
```

Comment:
since these names are language-dependent, consider using `locale`, reading `LC_*` env, or accepting a `lang` option to get those name in the desired locale?